compressor/management/commands/compress.py

```diff
@@ -211,44 +211,54 @@ def compress(self, log=None, **options):
                       "\n\t".join((t.template_name
                                    for t in compressor_nodes.keys())) + "\n")
 
+        contexts = settings.COMPRESS_OFFLINE_CONTEXT
+        if isinstance(contexts, six.string_types):
+            module, function = settings.COMPRESS_OFFLINE_CONTEXT.rsplit('.', 1)
+            contexts = getattr(import_module(module), function)()
```

Comment:
We should probably catch errors and re-raise as an `ImportError` with a message containing `settings.COMPRESS_OFFLINE_CONTEXT`, similar to how it's done in `compressor.cache` (see `get_cachekey`)